server/src/test/java/org/eclipse/lsp/cobol/usecases/TestDAFFileCloseStatement.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2021 DAF Trucks NV.
```

Comment:
You are using the old year here in new files. Can you please fix it?

Reply:
Done.